cmd/restic/cmd_sync.go

```diff
@@ -269,9 +295,69 @@ func runSync(opts SyncOptions, gopts GlobalOptions, term *termstatus.Terminal, a
 	if totalErrors > 0 {
 		Printf("There were %d errors\n", totalErrors)
 	}
+
+	postRestore := time.Now()
+	Verbosef("PostRestore time: %s\n", postRestore.Sub(postSnap))
 	p.Finish(id)
-	p.V("Files To Get: %v", fileTracker.FilesToGet)
-	p.V("Files To Delete: %v", fileTracker.FilesToDelete)
+
+	if VerboseLoggingEnabled() {
+		var filesFetched []string
+		for k, v := range fileTracker.FilesToGet {
+			if v == true {
+				filesFetched = append(filesFetched, k)
+			}
+		}
+		Verbosef("Files Fetched: %v\n", filesFetched)
+		Verbosef("Files To Delete: %v\n", fileTracker.FilesToDelete)
+		Verbosef("Directories To Delete: %v\n", fileTracker.UnexpectedDirs)
+		Verbosef("Total time: %s\n", time.Now())
+	}
+
+	if opts.Delete {
+		Verbosef("%d errors while deleting files\n", deleteFilesAndDirectories(fileTracker, opts))
```

Comment:
Can we put the deleteFilesAndDirectories on it's own line and then print out the amount?  That way if we are commenting out some logging, we don't loose the function call as well.

Reply:
Good spot, thanks!